at.ssh-mitm.server.json

```diff
@@ -0,0 +1,138 @@
+{
+    "app-id": "org.flatpak.Hello",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "ssh-mitm",
+    "finish-args": [
+        "--share=network"
+    ],
+    "modules": [
+        {
+            "name": "cpython",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.python.org/ftp/python/3.9.5/Python-3.9.5.tar.xz",
```

Comment:
do you really need 3.9.5?

3.8 is in the runtime, at least the one you should use (20.08)

Reply:
Thanks, i will use your python version. This makes it easier.